services/movie_session.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: str,
+                         movie_id: int,
+                         cinema_hall_id: int) -> None:
+    MovieSession.objects.create(show_time=movie_show_time,
+                                movie_id=movie_id,
+                                cinema_hall_id=cinema_hall_id)
+
+
+def get_movies_sessions(session_date: datetime.date = None) -> QuerySet:
+    sessions = MovieSession.objects.all()
+
+    if session_date:
+        sessions = sessions.filter(show_time__date=session_date)
+
+    return sessions
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def delete_movie_session_by_id(session_id: int) -> None:
+    MovieSession.objects.filter(id=session_id).delete()
+
+
+def update_movie_session(session_id: int,
+                         show_time: datetime = None,
+                         movie_id: int = None,
+                         cinema_hall_id: int = None) -> None:
+    session = MovieSession.objects.filter(id=session_id)

```

Comment:
the same

Reply:
why? i need "query" not "object" to use .update() i wanted to avoid working with object (than we need use save()) its complicates code. On my opinion.